src/main/java/org/spideruci/tacoco/MavenBuildProbe.java

```diff
@@ -103,14 +105,23 @@ public BuilderType getBuilderType() {
 	public String getClasspath(){
 		try{
 			if(classpath != null) return classpath;
-			if(!new File(targetDir+"/tacoco.cp").exists()) {
+            final String tacocoCpPath = new PathBuilder().path(targetDir).path("tacoco.cp").buildFilePath();
+
+			if(!new File(tacocoCpPath).exists()) {
                 MavenCli mavenCli = new MavenCli();
-                mavenCli.doMain(new String[] {"dependency:build-classpath", "-Dmdep.outputFile=tacoco.cp"}, targetDir,
+                mavenCli.doMain(new String[]{"dependency:build-classpath", "-Dmdep.outputFile=tacoco.cp"}, targetDir,
                         System.out, System.out);
 			}
-			classpath = new String(Files.readAllBytes(Paths.get(targetDir,"tacoco.cp")))
-					+":"+ targetDir + "/target/test-classes"
-					+":"+ targetDir + "/target/classes";
+
+            final String tacocoDependencies = new String(Files.readAllBytes(Paths.get(targetDir, "tacoco.cp")));
+            final String targetPath = new PathBuilder().path(targetDir).path("classes").buildFilePath();
```

Comment:
I did test it.But I used blinky as the system under test (I thoughts its safe enough for a sanity test). I guess I didn't get any error as Blinky didn't have any Junit test cases. I'll use another project from here onwards